Springboot/src/main/java/com/centralstudenthub/entity/Course.java

```diff
@@ -11,6 +15,8 @@
 @Getter
```

Comment:
Delete Getters and Setters. They are self-contained in @Data.

Reply:
Done!